fcrepo-auth-webac/src/test/resources/web.xml

```diff
@@ -29,6 +25,10 @@
       <listener-class>org.springframework.web.context.request.RequestContextListener</listener-class>
     </listener>
 
+    <listener>
```

Comment:
@peichman-umd :Also I just wanted to confirm whether this was tested manually using fcrepo-webapp (mvn jetty:run). I notice that the order of listeners and filters in the fcrepo-webapp web.xml is different from what you modified here. : https://github.com/fcrepo4/fcrepo4/blob/master/fcrepo-webapp/src/main/webapp/WEB-INF/web.xml

Reply:
@dbernstein I hadn't yet, but I can try it out today.

Reply:
@dbernstein I did discover that I need to look up nodes using the path and not the full URI. That should be fixed in my most recent commit on this branch.